src/VecSim/spaces/CMakeLists.txt

```diff
@@ -1,15 +1,14 @@
 
 # Build non optimized code in a single project without architecture optimization flag.
 project(VectorSimilaritySpaces_no_optimization)
-	add_library(VectorSimilaritySpaces_no_optimization STATIC
+	add_library(VectorSimilaritySpaces_no_optimization
 	L2/L2.cpp
 	IP/IP.cpp
 	)
 
-project(VectorSimilarity_Spaces)
 include(CheckCXXCompilerFlag)
 
-
+project(VectorSimilarity_Spaces)
 # TODO: Remove this once cpu_features get support for M1
 if(NOT APPLE)
```

Comment:
Wouldn't this be different on x86 apple?

Reply:
why?